src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/DefaultUnitTestingDocumentTrackingService.cs

```diff
@@ -1,31 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Immutable;
-using System.Composition;
-using Microsoft.CodeAnalysis.Host.Mef;
-
-namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.SolutionCrawler;
-
-[ExportWorkspaceService(typeof(IUnitTestingDocumentTrackingService), ServiceLayer.Default)]
-[Shared]
-internal sealed class DefaultUnitTestingDocumentTrackingService : IUnitTestingDocumentTrackingService
```

Comment:
the IUnitTestingDocumentTrackingService only ever had a single, default, no-op impl (this type).  Given that, it means this  type+interface was utterly pointless.  I removed it and unwound accordingly.